README.md

```diff
@@ -1,3 +1,16 @@
+# Wolf Motor
+The project will create the next breathtaking breakthrough in this planet's history.
```

Comment:
Beautiful.  Thanks for amplifying my meager muddling as a digital ditch digger.  To infinity and beyond....